lib/res_util/es_testdata.cpp

```diff
@@ -100,6 +106,51 @@ matrix_type * safe_copy(const matrix_type * m) {
 }
 
 
+matrix_type * matrix_delete_column(const matrix_type * m1, int column) {
```

Comment:
Is not the `libres` convention that these functions should have `alloc` in their names?